### PR TITLE
main/luanti: rename from minetest, update to 5.10.0

### PR DESCRIPTION
**Pull request by ttyyls**

### CODE REVIEW

File: main/luanti/template.py [hunk 1]
```diff
@@ -59,30 +60,31 @@
...
-    return ["usr/share/minetest/builtin"]
+    return ["usr/share/luanti/builtin"]
 
 
-@subpackage("minetest-server")
+@subpackage("luanti-server")
 def _(self):
     self.subdesc = "server"
-    self.depends = [self.with_pkgver("minetest-common")]
+    self.depends = [self.with_pkgver("luanti-common")]
```
Comment (q66) [inline main/luanti/template.py:88]: server is missing a provides...

Comment (ttyyls) [reply main/luanti/template.py:88]: the install process adds symlinks mintest->luanti and minetestserver->luantiserver. those get picked up by the pos-install hooks, adding them manually seemed redundant

Comment (q66) [reply main/luanti/template.py:88]: ? that doesn't make luanti-server package provide the minetest-server name (just like every other package in this template does with your changes)

Comment (ttyyls) [reply main/luanti/template.py:88]: true

**Pull request merged**